### ISSUE TITLE
Why is time_distributed_softmax gone?

### ISSUE THREAD

**Issue by philipperemy** [fchollet/keras#1889]
I'm wondering why this activation function has been removed?

https://github.com/patyork/keras/commit/465d8d14bc7d0b9e6c109cf2ff41e0aab1e627e7

```
model.add(TimeDistributedDense(len(chars)))
model.add(Activation('time_distributed_softmax'))
```

It's not that I particularly want to use it but I found an old script using Keras that was using that.

**Comment by farizrahman4u** [comment]
Just add a normal `softmax` activation. Keras will check your tensor shape and apply the right kind of softmax automatically.

```python
model.add(TimeDistributedDense(len(chars)))
model.add(Activation('softmax'))
```

**Comment by philipperemy** [comment]
@farizrahman4u Thank you so much!!

**Comment by philipperemy** [comment]
close #1889
